sms.py

```diff
@@ -1,24 +1,32 @@
-import dateutil.parser
 import glob
 import os
-import phonenumbers
 import re
 import time
-from datetime import datetime, timedelta
 from base64 import b64encode
-from bs4 import BeautifulSoup
+from datetime import datetime, timedelta
 from io import open  # adds emoji support
 from pathlib import Path
 from shutil import copyfileobj, move
 from tempfile import NamedTemporaryFile
 from time import strftime
 
+import dateutil.parser
+import phonenumbers
+from bs4 import BeautifulSoup
+
 sms_backup_filename = "./gvoice-all.xml"
 sms_backup_path = Path(sms_backup_filename)
 # Clear file if it already exists
 sms_backup_path.open("w").close()
 print("New file will be saved to " + sms_backup_filename)
 
+# Constant for allowed extensions
+VIDEO_EXTENSIONS = {'.3gp', '.mp4'}
+IMAGE_EXTENSIONS = {'.jpg', '.jpeg', '.png', '.gif', '.webp', '.heic'}

```

Comment:
I added heic and webp here, which were handled in some places, but not part of the allowed_extensions, the regex, or the file suffix list later in the file, so they would not have actually been used.